modules/store/testing/src/mock_store.ts

```diff
@@ -0,0 +1,39 @@
+import { Inject, Injectable } from '@angular/core';
+import { Observable, BehaviorSubject } from 'rxjs';
+import {
+  Action,
+  ActionsSubject,
+  INITIAL_STATE,
+  ReducerManager,
+  StateObservable,
+  Store,
+} from '@ngrx/store';
+
+@Injectable()
+export class MockStore<T> extends Store<T> {
+  private stateSubject = new BehaviorSubject<T>({} as T);
+  public scannedActions$: Observable<Action>;
+
+  constructor(
+    private state$: StateObservable,
```

Comment:
Type with `MockState<T>` instead of `StateObservable` and remove the additional type casting below